src/NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp

```diff
@@ -288,4 +288,43 @@ struct DerivCompute
       tmpl::list<VariablesTag, domain::Tags::Mesh<Dim>, InverseJacobianTag>;
 };
 
+/*!
+ * \ingroup DataBoxTagsGroup
+ * \brief Computes the spatial derivative of a single tensor tag not in a
+ * Variables.
+ *
+ * Computes the spatial derivative for a single tensor represented by
+ * 'TensorTag' in the frame mapped to by 'InverseJacobianTag'. It takes a
+ * single Tensor designated by 'TensorTag', the inverse Jacobian, and a mesh.
+ *
+ * This tag may be retrieved via `::Tags::DerivTensorCompute<TensorTag,
```

Comment:
Just remove this clarification as I requested before. This is just describing the standard way our compute tags work.